shipyard_blueprints/postgresql/shipyard_postgresql/cli/upload_file.py

```diff
@@ -65,182 +58,75 @@ def get_args():
-    return db_string
-
-
-def find_all_local_file_names(source_folder_name):
-    """
-    Returns a list of all files that exist in the current working directory,
-    filtered by source_folder_name if provided.
-    """
-    cwd = os.getcwd()
-    cwd_extension = os.path.normpath(f"{cwd}/{source_folder_name}/**")
-    file_names = glob.glob(cwd_extension, recursive=True)
-    return [file_name for file_name in file_names if os.path.isfile(file_name)]
-
-
-def find_all_file_matches(file_names, file_name_re):
-    """
-    Return a list of all file_names that matched the regular expression.
-    """
-    matching_file_names = []
-    for file in file_names:
-        if re.search(file_name_re, file):
-            matching_file_names.append(file)
-
-    return matching_file_names
-
-
-def combine_folder_and_file_name(folder_name, file_name):
-    """
-    Combine together the provided folder_name and file_name into one path variable.
-    """
-    combined_name = os.path.normpath(
-        f'{folder_name}{"/" if folder_name else ""}{file_name}'
-    )
-
-    return combined_name
-
-
-def determine_upload_method(db_connection):
-    if "db.bit.io" in str(db_connection):
-        upload_method = bitio_upload_method
-    else:
-        upload_method = "multi"
+            logger.info(f"{n_matches} files found. Preparing to upload...")
 
-    return upload_method
+            for index, key_name in enumerate(matching_file_names):
+                postgres.upload(
+                    key_name, table_name=table_name, insert_method=insert_method
+                )
+                if insert_method == "replace":
+                    insert_method = "append"
 
+            else:
+                logger.info(f"Successfully uploaded all files to {table_name}")
 
-def upload_data(source_full_path, table_name, insert_method, db_connection, schema):
-    upload_method = determine_upload_method(db_connection)
-    if os.path.getsize(source_full_path) < 50000000:
-        # Avoid chunksize if the file is small, since this is faster.
-        df = pd.read_csv(source_full_path)
-        df.to_sql(
-            table_name,
-            con=db_connection,
-            index=False,
-            if_exists=insert_method,
-            method=upload_method,
-            schema=schema,
-        )
-    else:
-        # Resort to chunks for larger files to avoid memory issues.
-        for index, chunk in enumerate(pd.read_csv(source_full_path, chunksize=10000)):
-            if insert_method == "replace" and index > 0:
-                # First chunk replaces the table, the following chunks
-                # append to the end.
-                insert_method = "append"
-
-            chunk.to_sql(
-                table_name,
-                con=db_connection,
-                index=False,
-                if_exists=insert_method,
-                method=upload_method,
-                chunksize=10000,
-                schema=schema,
+        else:
```

Comment:
I think its best to be explicit when possible
```
if match_type == exact_match:
elif match_type == regex
```

something i care less about but I have been doing in the scripts I refactored is moving the exact match logic before the regex since it doesn't have the nested logic that regex has.